cijoe/tests/tools/test_xnvme.py

```diff
@@ -236,3 +236,35 @@ def test_dsm(cijoe, device, be_opts, cli_args):
         f"xnvme dsm {cli_args} --nsid {device['nsid']} --ad --idw --idr --slba 0 --llb 1"
     )
     assert not err
+
+
+# For these three tests we request a device with labels 'bdev' and
+# "write_zeroes", as this will indirectly give us a kernel-managed NVMe device.
+@xnvme_parametrize(labels=["bdev", "write_zeroes"], opts=["be", "admin"])
+def test_subsystem_reset(cijoe, device, be_opts, cli_args):
+    err, _ = cijoe.run(f"xnvme subsystem-reset {device['uri']}")
+
+    if be_opts["be"] in ["linux"] and be_opts["admin"] in ["nvme"]:
+        assert not err, "Backend should support this, yet, yields an error"
+    else:
+        assert err, "Backend does not support this, yet, does not emit an error"
+
+
+@xnvme_parametrize(labels=["bdev", "write_zeroes"], opts=["be", "admin"])
+def test_subsystem_reset(cijoe, device, be_opts, cli_args):
```

Comment:
This test has the same name as the one above, as such we never actually test `subsystem-reset`